cmd/aries-js-worker/src/aries.js

```diff
@@ -183,11 +181,16 @@ const Aries = function(opts) {
             createInvitation: async function (text) {
                 return invoke(aw, pending,  this.pkgname, "CreateInvitation", text, "timeout while creating invitation")
             },
-            receiveInvitation: async function (text) {
-                return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", text, "timeout while receiving invitation")
+            receiveInvitation: async function (invitation) {
+                return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", invitation, "timeout while receiving invitation")
             },
-            acceptInvitation: async function (text) {
-                return invoke(aw, pending,  this.pkgname, "AcceptInvitation", text, "timeout while accepting invitation")
+            acceptInvitation: async function (connectionID) {
```

Comment:
The accept invitation API takes optional public DID as well (https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/command/didexchange/models.go#L86) along with connection ID.

Reply:
renamed to "id"

Reply:
ok: renamed back to `connectionID` and added an optional `publicDID` arg